jobs/test-override.Jenkinsfile

```diff
@@ -99,6 +106,22 @@ try {
         koji_build_ids += build_ids
     }
 
+    if (params.REPORT_TO_RESULTSDB) {
+        stage("Report Running") {
+            withCredentials([usernamePassword(credentialsId: 'resultsdb-auth',
+                                              usernameVariable: 'RDB_USERNAME',
+                                              passwordVariable: 'RDB_PASSWORD')]) {
+                shwrap("""/usr/lib/coreos-assembler/resultsdb-report    \
+                    --testcase cosa.build-and-test                      \
+                    --testcase-url ${JENKINS_URL}/job/test-override     \
+                    --testrun-url ${JENKINS_URL}/job/test-override      \
```

Comment:
Since we're now doing this from the `test-override` job itself, we can now directly pass the BlueOcean URL even here.

Reply:
will do